packages/volto-slate/locales/de/LC_MESSAGES/volto.po

```diff
@@ -0,0 +1,180 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: \n"
+"Language: \n"
+"Language-Team: \n"
+"Content-Type: \n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#: blocks/Title/TitleBlockEdit
+msgid "Add a description…"
+msgstr ""
+
+#: editor/plugins/SimpleLink/index
+msgid "Add link"
+msgstr "Link hinzufügen"
+
+#: blocks/Table/TableBlockEdit
+msgid "Align text"
+msgstr ""
+
+#: editor/plugins/StyleMenu/StyleMenu
```

Comment:
Left over from former StyleMenu. more below.

Reply:
@tiberiuichim @razvanMiu did we remove the StyleMenu? It seems there are this leftovers

Reply:
Updated.